app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -188,6 +189,7 @@ public static void init(Context context) {
             final boolean telemetryEnabled = isTelemetryEnabled(context);
 
             updateDefaultBrowserStatus(context);
+            updatePrefValue(context, resources.getString(R.string.pref_key_webview_version), DebugUtils.loadWebViewVersion(context));
```

Comment:
If we really care about performance, will it make sense to delay this update? We might benefit from using something like AlarmManager or any other scheduling mechanism. This will delay the update to the next onCreate() but I think it should be fine.

Reply:
However, I feel like to correctly evaluate the severity of performance issues, we'll need metrics to measure our performance which we don't have yet.